templates/Web/_composition/AngularJS/Page.Angular.AddRoute/src/app/app-routing.module_postaction.ts

```diff
@@ -0,0 +1,22 @@
+﻿import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+const routes: Routes = [
+//^^
```

Comment:
Don't check-in comments if you can avoid it.

Reply:
That is actually a macro used by the corets engine. It actually translates to "Put the following code (enclosed with //{[{ and //}]}) closer to the context given near the "//}]}" in this case it would insert the code closer to the "@NgModule" rather than the line "const routes: Routes". 

In other words it inserts the route at the bottom of the list rather than the top.

Reply:
As sahil mentioned, this is the format needed for CoreTS to merge two files together, not a regular comments

Reply:
Sorry, my lack of knowledge in this code.